src/components/Blocking/ItemListLarge.js

```diff
@@ -34,7 +34,7 @@ const styles = StyleSheet.create({
   container: {
     paddingVertical: 15,
     paddingHorizontal: 22,
-    borderTopColor: COLORS.lightgrey,
+    borderTopColor: COLORS.gray200,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The borderTopColor of the container style has been changed from lightgrey to gray200. This change seems to be consistent with the PR's purpose of changing colors. Please ensure that this color change does not affect the visibility and contrast of the border against its background.
```diff
- borderTopColor: COLORS.lightgrey,
+ borderTopColor: COLORS.gray200,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->